target/linux/generic/pending-5.4/483-mtd-spi-nor-add-esmt-f25l16pa.patch

```diff
@@ -0,0 +1,10 @@
+--- a/drivers/mtd/spi-nor/spi-nor.c
```

Comment:
Have you submitted this patch to upstream?

Reply:
I didn't... Should I summit this patch separately?

Reply:
No, not really, though you can send it to the upstream kernel as it's a rather simple patch. I've confused pending with backport, so I think it is fine at this state, please forget this.